### PR TITLE
Fixed Runset Stop-Continue Issues

### PR DESCRIPTION
**Pull request by arvindgh**

### CODE REVIEW

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5208675&bid=17081769)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5208675&bid=17081769)

File: Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs [hunk 1]
```diff
@@ -397,7 +397,10 @@ public override void SetReportRunSet(RunSetReport runSetReport, string logFolder
             ExecutionLoggerManager.RunSetReport.RunSetExecutionStatus = (eRunStatus)Enum.Parse(typeof(eRunStatus), runSet.RunStatus);
 
             SaveObjToReporsitory(runSet, liteDbManager.NameInDb<LiteDbRunSet>());
-            ExecutionLoggerManager.RunSetReport.liteDbRunnerList.Clear();
+            if (runSetReport.RunSetExecutionStatus != Amdocs.Ginger.CoreNET.Execution.eRunStatus.Stopped)
+            {
+                ExecutionLoggerManager.RunSetReport.liteDbRunnerList.Clear();
+            }
             ClearSeq();
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs]: Need to check if clear seq should be inside the condition

File: Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs [hunk 2]
```diff
@@ -208,7 +208,7 @@ public override object SetReportBusinessFlow(Context context, bool offlineMode,
             {
                 liteDbBFList.RemoveRange(0, context.Runner.BusinessFlows.Count);
             }
-            if (lastBfStatus == eRunStatus.Stopped && context.BusinessFlow.RunStatus != eRunStatus.Stopped)
+            if (lastBfStatus == eRunStatus.Stopped)
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs]: Test that their is not impact on the sequence after removing the condition

File: Ginger/GingerCoreNET/Logger/WebReportGenerator.cs [hunk 3]
```diff
@@ -215,12 +215,14 @@ private void PopulateMissingFields(LiteDbRunSet liteDbRunSet, string clientAppPa
                         }
                     }
                 }
+                liteDbRunner.RunStatus = liteDbRunner.SetStatus(liteDbRunner.BusinessFlowsColl);
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNET/Logger/WebReportGenerator.cs]: Need to check why the status is not updated during regular execution. We should not update it here explicitly

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5208675&bid=17081769)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5208675&bid=17081769)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5208675&bid=17081769)

**Pull request merged**